apps/docs/src/development/grid.md

```diff
@@ -17,6 +17,12 @@ The dynamic grid has the following advantages over a fixed column grid:
 - Ensures reasonable minimal and maximal size for individual elements
   :::
 
+Grid elements span the number of columns that is assigned to them.
+If there are less columns then an element is assigned, it will span all columns of the row.
+
+Additionally an element can be also be configured to span a specific column amount for a minium breakpoint (TODO: add link to UX breakpoint definition).
+Multiple span definition can then be combined to resize an element based on the breakpoint.
```

Comment:
```suggestion
Grid elements span the number of columns that is assigned to them.
If there are less columns available than an element is assigned, it will span all columns of the row.
All components already have a default span defined so you don't need to set anything if you just want to align with the default behavior.

Additionally, an element can also be configured to span a specific amount of columns for a minimum [breakpoint](/basics/breakpoints-grid).
Multiple span definitions can then be combined to resize an element based on the breakpoint.
```

FYI you need to merge the current main branch for the link to work :)

Reply:
I would like to only document things that we already do and have.

Reply:
Therefore I would like to leave out
> All components already have a default span defined so you don't need to set anything if you just want to align with the default behavior.

for now.

I created a ticket, so we don't forget about it: https://github.com/orgs/SchwarzIT/projects/5?pane=issue&itemId=51255861